src/soho/busyindicator/soho-busyindicator.directive.ts

```diff
@@ -105,14 +105,11 @@ export class SohoBusyIndicatorDirective implements AfterViewInit, AfterViewCheck
   /** Controls the activated state of the busy indicator. */
   @Input()
   public set activated(value: boolean) {
-    if (this.busyindicator) {
-      if (value) {
-        this.open();
-      } else {
-        this.close(true);
-      }
+    this.initiallyActive = value;
```

Comment:
Do we need this here?  I assumed this was just for the `ngAfterViewInit` method?

Reply:
Yes, I think so

Example scenario:

1. set activated true
2. ngAfterViewInit called
3. this.busyindicator is set
4. ngZone.run is called (with async callback)
5. set activated false (initiallyActive is still true, since this.busyindicator is truthy)
6. ngZone.run callback is called, which sets activated to initiallyActive, which is true)<
7. State is now "activated", even though it was set to false.

1 and 5 is performed by parent. Does that make sense?